manifests/repo.pp

```diff
@@ -13,7 +13,8 @@
         $location = $repo_location
         $description = 'MongoDB Custom Repository'
       } elsif $mongodb::globals::use_enterprise_repo == true {
-        $location = 'https://repo.mongodb.com/yum/redhat/$releasever/mongodb-enterprise/stable/$basearch/'
+        $mongover = split($version, '[.]')
```

Comment:
its used for all redhat family repos, not just enterprise. In the foss part of the decision tree, there is simple check if $version is set. I could add the same code to the enterprise check.